File: C_Sharp_code/Example/WindowsFormsApp/VB_demo04/Form1.vb
Public Class Form1
    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        Dim i As Integer = CInt(textBox1.Text)
        'Dim i As Integer = Convert.ToInt32(textBox1.Text)
        'Console.WriteLine("文數字轉數字:" + Convert.ToString(i))
        MessageBox.Show("文數字轉數字:" + CStr(i))
    End Sub
End Class

File: C_Sharp_code/Example/WindowsFormsApp/CS_demo04/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace CS_demo04
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            int i = Convert.ToInt32(textBox1.Text);
            //Console.WriteLine("文數字轉數字:"+i);
            MessageBox.Show("文數字轉數字:"+i.ToString());
        }

        private void textBox1_TextChanged(object sender, EventArgs e)
        {

        }
    }
}
